sktime/transformers/series_as_features/subsequence_transformer.py

```diff
@@ -0,0 +1,103 @@
+import numpy as np
+import pandas as pd
+import math
+from sktime.transformers.series_as_features.base \
+    import BaseSeriesAsFeaturesTransformer
+from sktime.utils.data_container import tabularize
+from sktime.utils.validation.series_as_features import check_X
+
+__author__ = "Vincent Nicholson"
+
+"""
+This class is to transform a univariate series into a
+multivariate one by extracting sets of subsequences.
+
+Proposed in the ShapeDTW algorithm.
+"""
+
+
+class SubsequenceTransformer(BaseSeriesAsFeaturesTransformer):
```

Comment:
is this different from the `IntervalSegmenter` or `RandomIntervalSegmenter` in `sktime/utils/transformer/series_as_features/segment.py`?

Reply:
Hello @mloning I tried adopting the IntervalSegmenter into ShapeDTW but ShapeDTW requires that you extract a window of say 5 around a data point, then move along by 1 then extract the next 5. I cannot see how to do this with IntervalSegmenter - I've attached a description on what im doing and what SubsequenceTransformer currently does :) 
![subsequence_transformer](https://user-images.githubusercontent.com/36476633/85870071-7e614b80-b7c4-11ea-8f0d-1dc78d92db5d.PNG)

Reply:
Okay, thanks for the explanation, yes that's something that the IntervalSegmenter does do. I'd it either call it something like `SlidingWindowSegmenter`  and put it in `transformers/series_as_features/segment.py` or make a private transformer only used in shapeDTW and call it `_SubsequenceTransformer`

Reply:
Oh okay that sounds like a good idea to me, yeah i'll do that :)